packit/utils.py

```diff
@@ -212,3 +200,23 @@ def assert_existence(obj):
     """
     if obj is None:
         raise PackitException("Object needs to have a value.")
+
+
+def graceful_get(d: dict, *keys, default=None) -> Any:
```

Comment:
`get()` is always 'graceful'. I'd suggest `nested_get` (not nasty) or `recursive_get`

Reply:
k, `nasty_get`